src/Compilers/CSharp/Portable/PublicAPI.Unshipped.txt

```diff
@@ -32,3 +32,5 @@ static Microsoft.CodeAnalysis.CSharp.SyntaxFactory.InterpolatedStringExpression(
 Microsoft.CodeAnalysis.CSharp.Conversion.IsUtf8StringLiteral.get -> bool
```

Comment:
>Utf8 [](http://example.com/codeflow?start=43&length=4)

Consider changing to `IsUTF8StringLiteral` for consistency with public API members below.